src/api/app/components/notification_action_description_component.rb

```diff
@@ -23,9 +25,9 @@ def call
         commentable = @notification.notifiable.commentable
         "#{commentable.project.name} / #{commentable.name}"
       when 'Event::RelationshipCreate'
-        "#{@user} made you #{@role} of #{@target_object}"
+        "#{@user} made #{@recipient} #{@role} of #{@target_object}"
```

Comment:
Probably out of this PR's scope but I was wondering if it would be worth the effort to have a link to the group here instead, just to let the user click and checkout the group members, reviews, requests, etc.

Orthogonal thinking it would also be nice to have a Group description text to define what is the scope or the purpose of that group, etc

Reply:
> Probably out of this PR's scope but I was wondering if it would be worth the effort to have a link to the group here instead, just to let the user click and checkout the group members, reviews, requests, etc.

It's a good idea, but I won't do this in this PR, since it could be done for the `@user` too and I want to keep the PR to the smallest set of changes possible.